packages/front-end/components/Experiment/AnalysisForm.tsx

```diff
@@ -320,6 +374,86 @@ const AnalysisForm: FC<{
           ]}
         />
       )}
+      {settings?.statsEngine === "frequentist" && (
```

Comment:
What if the org is using frequentist as the default, but the experiment is using Bayesian?

Reply:
There's no notion of experiment-level engine settings. Currently it's only set by org or report. (and soon, by project)